gemet/thesaurus/urls.py

```diff
@@ -81,19 +81,19 @@
         url(r'^inspire-themes/$', InspireThemesView.as_view(),
             name='inspire_themes'),
         url(r'^groups/$', GroupsView.as_view(), name='groups'),
-        url(r'^concept/(?P<concept_id>\d+)$', TermView.as_view(),
+        url(r'^concept/(?P<code>\d+)$', TermView.as_view(),
             name='concept'),
-        url(r'^inspire-theme/(?P<concept_id>\d+)$',
+        url(r'^inspire-theme/(?P<code>[a-zA-Z]+)$',
             InspireThemeView.as_view(), name='inspire_theme'),
-        url(r'^theme/(?P<concept_id>\d+)$', ThemeView.as_view(),
+        url(r'^theme/(?P<code>\d+)$', ThemeView.as_view(),
             name='theme'),
-        url(r'^group/(?P<concept_id>\d+)$', GroupView.as_view(),
+        url(r'^group/(?P<code>\d+)$', GroupView.as_view(),
             name='group'),
-        url(r'^supergroup/(?P<concept_id>\d+)$', SuperGroupView.as_view(),
+        url(r'^supergroup/(?P<code>\d+)$', SuperGroupView.as_view(),
             name='supergroup'),
-        url(r'^relations/(?P<group_id>\d+)/$', RelationsView.as_view(),
+        url(r'^relations/(?P<group_code>\d+)/$', RelationsView.as_view(),
```

Comment:
The URLs which shouldn't contain a trailing `/`are those that are referenced from the downloadable files. For `relations` and `theme_concepts` the URL can stay the way it was.